src/main/java/org/gridsuite/modification/server/modifications/LineAttachToVoltageLevel.java

```diff
@@ -32,12 +30,29 @@ public LineAttachToVoltageLevel(LineAttachToVoltageLevelInfos modificationInfos)
     }
 
     @Override
-    public void apply(Network network, Reporter subReporter) {
+    public void control(Network network) throws NetworkModificationException {
         LineCreationInfos attachmentLineInfos = modificationInfos.getAttachmentLine();
```

Comment:
Add existence check of lineToAttachToId ?

Reply:
this is checked at the beginning of apply()

Reply:
Yes, but i think that all possible checks should be done in the check method

Reply:
ok moved from apply()